README.md

```diff
@@ -24,6 +24,10 @@ Libp2p.<br>
 
 ### For Developers
 
+At build time the following env vars can be set to override default keys:
```

Comment:
maybe put a highlight here that the following env vars `must be set to override default keys during release process` ?